common/dbus_test.cpp

```diff
@@ -14,23 +14,59 @@
 
 #include <common/dbus.hpp>
 
+#include <stdlib.h> // setenv()
```

Comment:
Such comments tend to go out of date, or?

Reply:
Generally, yes, maybe. But this one is quite special, I think. Including `stdlib.h` when there is `cstdlib` is a bit weird, but `setenv()` just doesn't exist in the latter.

Reply:
Ah, right, I didn't even think about `cstdlib` in this instance (but I should have!). Maybe put a slightly bigger comment then, on why `cstdlib` can't be used.